src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinInventoryEffectRenderer_FixPotionEffectNumerals.java

```diff
@@ -0,0 +1,42 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import com.mitchej123.hodgepodge.Common;
+import com.mitchej123.hodgepodge.util.RomanNumerals;
+import net.minecraft.client.renderer.InventoryEffectRenderer;
+import net.minecraft.potion.PotionEffect;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+@Mixin(InventoryEffectRenderer.class)
+public class MixinInventoryEffectRenderer_FixPotionEffectNumerals {
+
+    private int hodgepodge$potionAmplifierLevel;
+
+    @Redirect(
+            method = "func_147044_g",
+            at = @At(value = "INVOKE", target = "Lnet/minecraft/potion/PotionEffect;getAmplifier()I", ordinal = 0))
+    public int hodgepodge$skipOriginalCode(PotionEffect potionEffect) {
+        this.hodgepodge$potionAmplifierLevel = potionEffect.getAmplifier();
+        return 1;
```

Comment:
To keep the current behaviour of not showing the number in cases like "Speed I" or "Regeneration I":
`return Math.min(1, this.hodgepodge$potionAmplifierLevel);`

Reply:
I am already doing that in the second redirect where I check if (this.hodgepodge$potionAmplifierLevel > 0) and return an empty string if not. For you know a potionEffect.getAmplifier() = 0 means an effect like Speed 1. It's off by one

Reply:
Sorry, missed that. What I wrote one comment below still holds though and it would make it easier there if you also changed here what I suggested.